app/Services/GitHub/GitHubSyncCoordinator.php

```
@@ -0,0 +1,38 @@
+use App\Services\GitHub\Connection\GitHubConnectionState;
+use App\Services\GitHub\Connection\GitHubConnectionStatus;
+use Illuminate\Contracts\Bus\Dispatcher;
+
+class GitHubSyncCoordinator
+{
+    public function __construct(
+        private GitHubConnectionManager $connections,
+        private Dispatcher $dispatcher,
+    ) {}
+
+    public function requestSync(?string $personalAccessToken = null, bool $force = false): GitHubSyncDispatchResult
```

PHPMD (2.15.0)

The method requestSync has a boolean flag argument $force, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)